docs/content/en/docs/tutorials/debugging-tetragon.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Debugging Tetragon"
+weight: 1
+icon: "overview"
+description: "Diagnosing Tetragon problems"
+---
+
+### Enable debug log level
+
+Debugging problems is easy if we have the right logs. To change Tetragon log level use the `log-level` option:
```

Comment:
How about: "When debugging, it might be useful to change the log level. The default log level is controlled by the `log-level` option: